### PR TITLE
filter config values on testSuiteStart

### PR DESCRIPTION
**Pull request by bcerban**
### Fixed Issues (if relevant)

1. magento/magento2#22370: Filtering ignored config values in test framework causes error

### Manual testing scenarios (*)

1. Set `compiled_config => 1` in `app/etc/env.php` cache types. 
2. Then run the integration test suite. 

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function startTestSuite()
     {
         if (null === $this->reader) {
             $this->reader = Bootstrap::getObjectManager()->get(\Magento\Framework\App\DeploymentConfig\Reader::class);
-            $this->config = $this->reader->load();
+            $this->config = $this->filterIgnoredConfigValues($this->reader->load());
```
Comment (aleron75) [inline dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php:54]: Hello, maybe I'm missing something but where the `filterIgnoredConfigValues()` method comes from?

Comment (bcerban) [reply dev/tests/integration/framework/Magento/TestFramework/Isolation/DeploymentConfig.php:54]: Hi @aleron75 it's on line 90 in that same file. It is only available on the 2.2 releases, and was included in version 2.2.8.

**Review by aleron75 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @aleron75, thank you for the review.
[ENGCOM-4814](https://jira.corp.magento.com/browse/ENGCOM-4814) has been created to process this Pull Request

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed

**Before:**

before

**After:**

after

**Pull request merged**